tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/AspireEFMySqlExtensionsTests.cs

```diff
@@ -81,14 +95,14 @@ public void CanConfigureDbContextOptions()
     {
         var builder = Host.CreateEmptyApplicationBuilder(null);
         builder.Configuration.AddInMemoryCollection([
-            new KeyValuePair<string, string?>("Aspire:Pomelo:EntityFrameworkCore:MySql:ServerVersion", "8.2.0-mysql"),
+            new KeyValuePair<string, string?>("Aspire:Pomelo:EntityFrameworkCore:MySql:ServerVersion", _serverVersion),
             new KeyValuePair<string, string?>("ConnectionStrings:mysql", ConnectionString),
             new KeyValuePair<string, string?>("Aspire:Pomelo:EntityFrameworkCore:MySql:Retry", "true")
         ]);
 
         builder.AddMySqlDbContext<TestDbContext>("mysql", configureDbContextOptions: optionsBuilder =>
         {
-            optionsBuilder.UseMySql(new MySqlServerVersion(new Version(8, 2, 0)), mySqlBuilder =>
+            optionsBuilder.UseMySql(new MySqlServerVersion(new Version(MySqlContainerImageTags.Tag)), mySqlBuilder =>
```

Comment:
It might make sense to pull `new MySqlServerVersion(new Version(MySqlContainerImageTags.Tag))` into a single location so all these tests don't need to duplicate it.